tests/jit/test_bgmv_moe_jit_cache.py

```
@@ -0,0 +1,121 @@
+    "kernel_config.h",
+]
+
+# A fixed timestamp far in the past (2020-09-13T12:26:40Z), so that any staging
+# step that stamps "now" onto a copy is unambiguously detectable without sleeps.
+_OLD_MTIME_NS = 1_600_000_000_000_000_000
+
+
+class _StubCompilationContext:
+    """No-GPU stand-in so the generation pass needs no CUDA device."""
+
+    TARGET_CUDA_ARCHS = [(9, "0a")]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)